### PR TITLE
[10.5.0] Update translations

### PR DESCRIPTION
**Pull request by phil-davis**
## Description
"backport" the Transifex changes in `master` from the last 2 days into the `release-10.5.0` branch.

This gets the translations up-to-date. In particular some translations for "Manual File Locking" and "Login"

Note: no changelog needed - we do not do changelogs for commits that came across from Transifex

## How Has This Been Tested?
CI

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: core/l10n/bs.json [hunk 1]
```diff
@@ -152,6 +152,7 @@
     "Search" : "Potraži",
     "Server side authentication failed!" : "Autentikacija na strani servera nije uspjela!",
     "Please contact your administrator." : "Molim kontaktirajte svog administratora.",
+    "Login" : "Prijava",
```
Comment (DeepDiver1975) [inline core/l10n/bs.json:155]: There is something wrong.
We have login an log in.
This should be consistent.

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]

**Comment by DeepDiver1975** [comment]
Strange that these commits are done by me .... Should be owncloud account

File: core/l10n/bs.json (hunk 1 above)
Comment (phil-davis) [reply core/l10n/bs.json:155]: There are existing sentences like:
`core/templates/login.php`

```
You are trying to access a private link. Please log in first.
```
And
```
			<input type="submit" id="submit" class="login primary icon-confirm" title="<?php p($l->t('Log in')); ?>" value="" disabled="disabled"/>
```
which is the hover text of the "right-arrow" icon that you press to "log in" or "login".

These are existing things.

IMO the translations are already in core `master`. So we need to get them into `release-10.5.0` anyway.

And as a separate issue, we need to decide if it should be "login" or "log in" in each place.

**Comment by phil-davis** [comment]
> Strange that these commits are done by me .... Should be owncloud account

The Transifex commit has been generated with your name for "a long time"  (tm)

File: core/l10n/bs.json (hunk 1 above)
Comment (phil-davis) [reply core/l10n/bs.json:155]: I raised issue #37603

**Comment by DeepDiver1975** [comment]
> The Transifex commit has been generated with your name for "a long time" (tm)

no - there has been a change recently - see e.g. https://github.com/owncloud/core/commits/632544afb539dbd66906522efd8f5fd978f29b2b/core/l10n/af_ZA.js

File: core/l10n/bs.json (hunk 1 above)
Comment (jnweiger) [reply core/l10n/bs.json:155]: The verb in `Please log in first` and the noun `Login` should be spelled exactly like seen above. One with a whitespace, the other without.

Comment (DeepDiver1975) [reply core/l10n/bs.json:155]: > You are trying to access a private link. Please log in first.

this is not the string we see in the translations files. There must we a single string 'Log in' ...

**Comment by phil-davis** [comment]
@DeepDiver1975 see https://github.com/owncloud/core/issues/37603#issuecomment-650053201

The "new" "Login" form seems to be preferred to that other string "Log in", so actually I will probably make a PR to change that "Log in" to "Login". That means we do need the string "Login" translated.

So, IMO, this translation PR  to `release-10.5.0` can go ahead. (And more translation changes will come every day anyway)

Please review again.

File: core/l10n/bs.json (hunk 1 above)
Comment (phil-davis) [reply core/l10n/bs.json:155]: "Log in" is hover-text on the "submit" button, which is the right-arrow icon on the login page.
"Login" is the placeholder text in the "username" field on the login page when `strict_login_enforced` is set.

**Review by DeepDiver1975 — APPROVED** [review]

**Pull request merged**